_posts/2020-08-24-gsoc-report-cbouy.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "GSoC Report: From RDKit to the Universe and back 🚀"
+---
+
+With the end of the summer comes the end of my awesome Google Summer of Code adventure with the MDAnalysis team. It's also the occasion for me to report on the code I've implemented, the things I've learned and the challenges I've faced.
+
+## Summary of the project
```

Comment:
One paragraph of motivation would be good. *Why* are you doing this. What were we able to do before, what was missing, what can we do now? You can recycle from your proposal.